samples/snippets/README.md

```diff
@@ -0,0 +1,29 @@
+# Google Retail Search
+## Java code samples
+The code here demonstartes how to consume Google Retail Search API in Java
+
+### Authorization
+In order to authenticate and authorize the client place a JSON file with authorization token to your cloudshell project and refer environment variable `GOOGLE_APPLICATION_CREDENTIALS` to it (use full path to file):
+
+```
+export GOOGLE_APPLICATION_CREDENTIALS="/home/use_name/cloudshell_open/grs-retail/token.json"
```

Comment:
Generally we encourage folks to just use [Application Default Credentials (ADC)](https://cloud.google.com/docs/authentication/best-practices-applications)

Reply:
It is recommended for Retail clients to authenticate using GOOGLE_APPLICATION_CREDENTIALS